### PR TITLE
Add sync_scroll

### PR DESCRIPTION
**Pull request by andy-takker**
- Add shortcut for using one pager for two scrollable widgets
- Add example for `sync_scroll`

### CODE REVIEW

File: src/aiogram_dialog/utils.py [hunk 1]
```diff
@@ -174,3 +176,18 @@ def remove_indent_id(callback_data: str) -> Tuple[Optional[str], str]:
         intent_id, new_data = callback_data.split(CB_SEP, maxsplit=1)
         return intent_id, new_data
     return None, callback_data
+
+
+def sync_scroll(scroll_id: str) -> Callable[
```
Comment (Tishka17) [inline src/aiogram_dialog/utils.py:181]: I guess we could support multiple ids here.

Additionally, as it is used as `on_page_changed` the user will loose ability set own callback, so we can support it here.

Comment (Tishka17) [inline src/aiogram_dialog/utils.py:181]: Probably, we should to put this function to widgets next to BaseScroll, not in common utils.

**Comment by andy-takker** [comment]
@Tishka17 updated

**Review by Tishka17 — APPROVED** [review]

File: src/aiogram_dialog/widgets/common/scroll.py [hunk 2]
```diff
@@ -75,3 +75,24 @@ async def set_page(
 
     def managed(self, manager: DialogManager) -> ManagedScroll:
         return ManagedScroll(self, manager)
+
+
+def sync_scroll(
```
[the remark this answers is not in the document]
Comment (andy-takker) [reply src/aiogram_dialog/widgets/common/scroll.py:80]: Added and updated the example

**Review by Tishka17 — APPROVED** [review]

**Pull request merged**